tests/simulator/advance/test_fields_advance_3d.py

```
@@ -0,0 +1,124 @@
+    @unpack
+    def test_overlaped_fields_are_equal_with_min_max_patch_size_of_max_ghosts(
+        self, interp_order, refinement_boxes
+    ):
+        print(f"{self._testMethodName}_{ndim}d")
+        time_step_nbr = 3
+        time_step = 0.001
+        from pyphare.pharein.simulation import check_patch_size
+
+        cells = [18] * ndim
+
+        largest_patch_size, smallest_patch_size = check_patch_size(
```

Ruff (0.15.2)

[warning] Unpacked variable `largest_patch_size` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)